content/docs/apps/govcloud.md

```diff
@@ -28,3 +28,33 @@ When your org starts using cloud.gov's GovCloud environment, here are changes to
 ### Tips
 
 - If you need to work in both the East/West and GovCloud environments, [this Cloud Foundry plugin](https://github.com/guidowb/cf-targets-plugin) may be helpful.
+
+### Migration
+
+_This section is a work in progress._
+
+As part of [cloud.gov achieving FedRAMP authorization](https://18f.gsa.gov/2016/07/18/cloud-gov-full-steam-ahead-fedramp-assessment-process/), all tenants in the AWS East/West will need to move to the new GovCloud environment.
+
+#### Who this affects
+
+Any systems running in the AWS East/West version of cloud.gov.
```

Comment:
Can you add a note about how to check for this? E.g. all apps / spaces you see when going to dashboard.cloud.gov are impacted. Or provide command line instructions for how to check.